src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -191,12 +191,12 @@ def getElementsStatusAndJobsByWorkflow(self, inboxFlag=False, stale=True):
         options = {'reduce' : True, 'group_level': 2}
```

Comment:
If group_level is set group is always true.   I will change the function name. :-)